config.yaml

```diff
@@ -174,11 +174,10 @@ rabbitmq:
     cloudify_rabbitmq_package: cloudify-rabbitmq-*.noarch.rpm
 
   # Sets the File Descriptor limit for the rabbitmq user.
-  # Not used if an external endpoint is used.
```

Comment:
This isn't relevant any more because it's all under the tombstone that gives this information.